awgbuild.sh

```diff
@@ -5,10 +5,10 @@ export REACT_APP_SLIDE_IMAGE_ENDPOINT="/auth/api/v0/tile/"
 export REACT_APP_COMMIT_HASH=`git rev-parse --short HEAD`
 export REACT_APP_COMMIT_TAG=`git tag -l --points-at HEAD`
 export REACT_APP_API="https://portal.awg.gdc.cancer.gov/auth/api"
+export REACT_APP_FENCE="https://portal.awg.gdc.cancer.gov/fence/"
 export REACT_APP_GDC_AUTH="https://portal.awg.gdc.cancer.gov/auth/"
-export REACT_APP_FENCE="https://login.awg.gdc.cancer.gov"
+# export AWG_TOKEN_EXPIRY=5
```

Comment:
This flag sets a custom number of minutes the UI should wait before allowing another query to renew the token.
(Default is 5, hardcoded)